qa/rpc-tests/sc_invalidate.py

```diff
@@ -215,6 +199,7 @@ def run_test(self):
 
         mark_logs("\nJoining network", self.nodes, DEBUG_MODE)
         self.join_network()
+        time.sleep(2)
```

Comment:
Would it be possible to use the new `join_network_sync()` instead of `time.sleep()` here?
```suggestion
        self.join_network_sync()
```

Reply:
Moved to ZEND-616](https://horizenlabs.atlassian.net/browse/ZEND-616)